src/main/java/com/playmonumenta/networkrelay/GatherRemotePlayerDataEvent.java

```diff
@@ -0,0 +1,59 @@
+package com.playmonumenta.networkrelay;
+
+import com.google.gson.JsonObject;
+import java.util.HashMap;
+import java.util.Map;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Can you remove all the instances of NotNull? This is the default, if it's not @Nullable then it's automatically assumed to be NotNull.

Reply:
Was automatically added by intelliJ
![image](https://github.com/TeamMonumenta/monumenta-network-relay/assets/78341107/cc7a6261-4f35-4e83-8e94-8bc65a6519d8)